docs/cli.md

```diff
@@ -27,7 +27,7 @@ You can also find information on flags with `promscale_<version> -help`.
 | read-only | boolean | false | Read-only mode for the connector. Operations related to writing or updating the database are disallowed. It is used when pointing the connector to a TimescaleDB read replica. |
 | use-schema-version-lease | boolean | true | Use schema version lease to prevent race conditions during migration. |
 | async-acks | boolean | false | Acknowledge asynchronous inserts. If this is true, the inserter will not wait after insertion of metric data in the database. This increases throughput at the cost of a small chance of data loss. |
-| tput-report | duration | 1 second | Interval in seconds at which throughput should be reported. Setting it to `0` will disable reporting throughput. |
+| tput-report | duration | 1 second | Setting duration to `0` will disable reporting throughput, else interval with unit at which throughput should be reported, e.g. `10s` or `3m`. |
```

Comment:
```suggestion
| tput-report | duration | 1 second | Duration interval at which throughput should be reported. Setting duration to `0` will disable reporting throughput, otherwise, an interval with unit must be provided, e.g. `10s` or `3m`. |
```